index.html

```diff
@@ -39,8 +39,205 @@ <h1 class="hero-title" >I’m Mariane <br>
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+                <!-- work 4  -->
+            <article class="work ">
+                <img class="work-main-img" src="./assets/Snapshoot Portfolio4.svg" alt="">
+                <aside class="work-info">
+                    <h2 class="work-title" >Multi-Post Stories</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about ">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
+                <p class="hero-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <a href="#">LET'S CONNECT</a>
+
+                <aside class="social-media">
+                    
+                    <a href="">
+                        <img src="./assets/social media icons.svg" alt="">
+                    </a>
+                </aside>
```

Comment:
- [ ] Kindly add the right color to the `LET'S CONNECT` text. It currently looks different from the Figma design and also please ensure that you are using all the right design guidelines including font-size, font-weigth, etc.
- [ ] Please ensure that you have applied the right margin and padding styles in your design. Currently, your design doesn't have enough spacing compared to the Figma design so kindly fix this as well :+1: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/219580068-61c011f3-01a1-4028-a848-ff6af435c310.png)
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/219580220-a767e71c-f49c-46ce-9159-8f1b6d457e7e.png)